cirq/linalg/decompositions.py

```diff
@@ -264,6 +264,110 @@ def so4_to_magic_su2s(
     return a, b
 
 
+@value.value_equality(approximate=True)
+class AxisAngleDecomposition:
+    """Represents a unitary operation as an axis, angle, and global phase."""
```

Comment:
Give an explicit unitary that this corresponds to, i.e. what unitary produces.  Because we want to make sure we get the +\- consistent.

Reply:
I added one. Is it the one you expected?

(Please check factors of 2 and such..)

Reply:
Where?

Reply:
Oh, I added it to the unitary method. I guess it does make more sense on the class.

Reply:
Moved.